### ISSUE TITLE
Windows 10下ttc版本的字体会有乱码

### ISSUE THREAD

**Issue by Measureless** [be5invis/Sarasa-Gothic#53]
![snipaste_2018-10-16_20-03-19](https://user-images.githubusercontent.com/30423783/47015245-0e8d6780-d17f-11e8-92e1-a32c3f71b0da.png)
![snipaste_2018-10-16_20-03-44](https://user-images.githubusercontent.com/30423783/47015247-0f25fe00-d17f-11e8-8166-87aeeb10eb28.png)
![snipaste_2018-10-16_20-04-17](https://user-images.githubusercontent.com/30423783/47015248-0f25fe00-d17f-11e8-816a-230946118062.png)

**Comment by be5invis** [comment]
Have you tried to uninstall before installing newer version?

**Comment by Measureless** [comment]
> Have you tried to uninstall before installing newer version?

Yes, I uninstalled the ttf version (0.60) before installing ttc version (0.60).

**Comment by be5invis** [comment]
Are you using MACTYPE or something like that?
If so, restart or disable it since it may unable to handle edge cases of TTC files.
The TTC files are verified with OTS and do not have any problem internally with its encoding tables.

**Comment by Measureless** [comment]
@be5invis Thank you very much!

**Comment by Measureless** [comment]
Figured out. All I need to do is to reboot the system. I just encountered the same problem when I use Version 0.7.0 ttf. ┓( ´∀` )┏
